hw2/03-count.js

```diff
@@ -1,3 +1,34 @@
-// Add your code here
+const input = document.querySelector('input');
 
 input.addEventListener('keydown', handleKeyDown);
+
+function handleKeyDown(event) {
+  if (event.key == 'Enter') {
+    const inputString = event.target.value.trim();
```

Comment:
good job here on efficiency I wanted to modify my code to also use the target value but haven't gotten around to it but this is a nice and clean approach. good job.